sdk/examples/churn_prediction/Build and Deploy Churn Ensemble.ipynb

```diff
@@ -712,7 +712,7 @@
    "outputs": [],
```

Comment:
<p>"An Aqueduct <code>check</code>"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aqueducthq/aqueduct/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
Resolved. Can you accept?